.github/workflows/build-cirros-bionic.yaml

```diff
@@ -0,0 +1,189 @@
+name: CirrOS image builder
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build-cirros-aarch64:
+    name: aarch64
+    runs-on: ubuntu-18.04
```

Comment:
I would go for 20.04 for now. @osfrickler and I have worked on updating to newer Buildroot and newer Ubuntu kernels so 22.04 will be final version.

Reply:
@hrw I saw that you merged a new update for CirrOS. Does that contain the necessary updates for building on Focal instead of Bionic?

If so, I can merge that commit into this pull request so that we do not accidentally cross the streams.

Reply:
Those changes does not affect your PR.